security/firely-auth/firely-auth-sso.rst

```diff
@@ -63,6 +63,10 @@ Multiple configuration parts are necessary to enable SSO in Firely Auth:
     Based on the ``AutoProvisionFromSecurityGroup`` setting it is possible to restrict the sign-up of users based on security groups defined in the SSO provider. The attribution of a user account to a one or more security group needs to be exposed via the ``groups`` claim.
     If the ID token received from the SSO provider contains such a claim and the value is part of the whitelisted security groups in the appsetttings, the auto-provisioning is allowed by Firely Auth. Note that Azure allows you to set different values for this claim, such as the Object ID or the display name of the security group. Depending on how this claim is configured in Azure, the respective value, either Object ID or display name of the Security Group, should be added to the ``AutoProvisionFromSecurityGroup`` list to allow auto-provisioning for this group.
 
+.. admonition:: Docker deployments
+
+    If :ref:`testing with Docker <firely_auth_deploy_docker>`, Firely Auth should be configured to use the local host network, instead of the default bridge network. This is necessary to allow the SSO provider to redirect back to the Firely Auth instance running in a Docker container.
```

Comment:
```suggestion
    If :ref:`testing with Docker <firely_auth_deploy_docker>`, Firely Auth should be configured to use the localhost network, instead of the default bridge network. This is necessary to allow the SSO provider to redirect back to the Firely Auth instance running in a Docker container.
```
```suggestion
    If :ref:`testing with Docker <firely_auth_deploy_docker>`, Firely Auth should be configured to use the local host network, instead of the default bridge network. This is necessary to allow the SSO provider to redirect back to the Firely Auth instance running in a Docker container.
```